pkg/model/gcemodel/api_loadbalancer.go

```diff
@@ -99,6 +99,18 @@ func createPublicLB(b *APILoadBalancerBuilder, c *fi.CloudupModelBuilderContext)
 			TargetTags:   []string{b.GCETagForRole(kops.InstanceGroupRoleControlPlane)},
 			Allowed:      []string{"tcp:" + strconv.Itoa(wellknownports.KubeAPIServer)},
 		})
+
+		if b.NetworkingIsIPAlias() {
+			c.AddTask(&gcetasks.FirewallRule{
```

Comment:
Presumably this is to allow traffic from the LB to the control plane nodes.

Do we need a rule to allow the PodCIDRs to get traffic to the internal load balancer?

Reply:
We don't have a rule for the nodes, so I'm assuming not.  I think the internal load balancer is open to anything on the subnets.  I can try sending another PR to lock it down to the source tags (for node IPs) + the pod CIDR (source tags don't work for secondary IPs), but we don't have a test for it either at the moment, I believe, so likely another PR...